src/Core/Content/ProductExport/Error/XmlValidationError.php

```diff
@@ -7,26 +7,17 @@
 #[Package('inventory')]
 class XmlValidationError extends Error
 {
-    /**
-     * @var string
-     *
-     * @deprecated tag:v6.7.0 - Will be natively typed
-     */
-    protected $id;
+    protected string $id;
```

Comment:
Nitpick, but when you need to update the branch anyway: $id and $errors can be declared in the constructor directly